docs/install-faq.rst

```diff
@@ -1,7 +1,6 @@
 * I cannot install the package and it reports the error "XXX is not a supported wheel on this platform".
```

Comment:
Install-FAQ should clarify that MXNet version 1.7 is needed

Reply:
Install-FAQ should clarify these messages are Mac-specific and only related to installation of custom MXNet wheel:

I cannot install the package and it reports the error "XXX is not a supported wheel on this platform".
I see the error "ERROR: No matching distribution found for mxnet<2.0.0,>=1.7.0b20200713".

Reply:
This is not a Mac specific error

> Install-FAQ should clarify these messages are Mac-specific and only related to installation of custom MXNet wheel:
> 
> I cannot install the package and it reports the error "XXX is not a supported wheel on this platform".
> I see the error "ERROR: No matching distribution found for mxnet<2.0.0,>=1.7.0b20200713".

The error can also occur on Linux with outdated pip.

Reply:
Note that is PR is updating the install-instructions for Linux to simply be: `pip install mxnet`.
The wheel-error can still occur with this command?  

I figured the main error Linux users will now face (after this PR) is their pip has installed MXNet version <= 1.6 rather than the required 1.7.  So may be good to address this new potential issue separately in the FAQ.